templates/components/progress-bar.html.twig

```diff
@@ -0,0 +1,51 @@
+{% macro progressBars(items, options) %}
+
+    {% set _progressSize = options.progressSize ?? '' %}
+    {% set _backgroundColor = options.backgroundColor ?? '' %}
+    {% set _style = options.style ?? '' %}
+
+    {% set _items = items ?? [] %}
+
+    <div class="progress {{ _backgroundColor }} {{ _progressSize }}" style="{{ _style }}">
+        {% for item in _items %}
+
+            {% set _value = item.current ?? 0 %}
+            {% set _max = item.max ?? 0 %}
+            {% set _min = item.min ?? 0 %}
+            {% set _indeterminate = item.indeterminate ?? _value == _max and _min == _max %}
+            {% set percent = _indeterminate or (_max - _min) == 0 ? 0 : ((_value - _min) * 100) / (_max - _min) %}
```

Comment:
not that I care deeply, but to be consequent shouldn't that have an underscore as well?

Reply:
`percent` is the only var that is not coming from user params. It is a calculated value.
So no `_` for him.

Do you want me to add it ?

Reply:
Fine with me. But we should write all of our own rules down at some place.